docs/tutorials/integrations/kserve/http.md

```diff
@@ -76,7 +75,7 @@ Open Grafana by going to [http://localhost:3000](http://localhost:3000).
 [Add a JSON API data source](http://localhost:3000/connections/datasources/marcusolsson-json-datasource) `Iter8` with the following parameters:
```

Comment:
This data source is still named `Iter8`. Did we want to change this to match the test name?